rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1107,14 +1108,21 @@ std::vector<rocalTensorList *> MasterGraph::create_coco_meta_data_reader(const c
         default_mask_info.set_metadata();
         _meta_data_buffer_size.emplace_back(_user_batch_size * default_mask_info.data_size());
     }
+    if (metadata_type == MetaDataType::PixelwiseMask)
```

Comment:
Can this be and else if?

Reply:
Polygon info required for calculating pixelwiselabels. It can be changes to else if I think